product_docs/docs/pgd/5/reference/nodes.mdx

```diff
@@ -17,7 +17,7 @@ indexdepth: 3
 -   `CATCHUP`: Initial sync phase is completed. Now the join is at the last step
     of retrieving and applying transactions that were performed on the upstream
     peer node since the join started.
--   `STANDBY`:  Node join finished, but hasn't yet started to broadcast changes.
+-   `STANDBY`:  Node join finished but hasn't yet started to broadcast changes.
     All joins spend some time in this state, but if defined as a logical standby,
     the node continues in this state.
 -   `PROMOTE`: Node was a logical standby and we just called `bdr.promote_node` to
```

Comment:
Wondering about the use of "we" here. Could this be changed to "you"?

Reply:
Changed to "and `bdr.promote_node` was just called to...`